ansible-ipi-install/roles/installer/defaults/main.yml

```diff
@@ -2,3 +2,46 @@
 # defaults file for installer
 cache_enabled: false
 provision_cache_store: "{{ ansible_env.HOME }}/rhcos_image_cache/"
+
+# Information used to create certificates when creating the registry.
+cert_country: MyCountry
+cert_state: MyState
+cert_locality: MyCity
+cert_organization: MyCompany
+cert_origanizational_unit: MyBusinessUnit
+
+# A file that contained extra auths to be included in the pull secret.
+disconnected_registry_auths_file:
+
+# Name of a file containing the additionalTrustBundle and mirrored
+#   repository settings. If this is set, then a local registry will not be created.
+#   Setting this causes the playbook to use an existing disconnected registry.
+disconnected_registry_mirrors_file:
+
+# The containers registry port
+registry_port_container: 5000
+
+# The port exposed on the server for the registry
+registry_port: "{{ registry_port_container }}"
+
+# The following variables define the registry location
+registry_dir: /opt/registry
```

Comment:
instead of /opt, can we change this to: {{ ansible_user_dir }}/registry

Reply:
these tasks allow the deployment of a registry on a different server. possible one acting as a disconnected registry for other systems in the future. not sure ansible_user_dir would be the best option in that case.

Reply:
Thats a good point. However, we could also add this in the inventory/hosts.sample file with something like
```
# Set the disconnected registry path location. Default is /opt/registry. Uncomment to set a different value. 
#registry_dir="/opt/registry"
```

Reply:
I will add something to the hosts.sample file showing this as a changeable option.

Reply:
added and will be pushed with next commit

Reply:
can we remove empty line in defaults/main.yml? also see comment regarding comments in defaults/main.yml